fuzz/corpora/cms/2aba8037213156ea1593054768d4576cb8d08309

```diff
@@ -0,0 +1 @@
+0	*�H��

```

Comment:
extra blankline on this file and the next one

Reply:
This is a fuzz corpora binary file

Reply:
these are malformed CMS_ContentInfo with missing envelopedData
$ apps/openssl asn1parse -inform der -in fuzz/corpora/cms/2aba8037213156ea1593054768d4576cb8d08309
    0:d=0  hl=2 l=  11 cons: SEQUENCE
    2:d=1  hl=2 l=   9 prim: OBJECT            :pkcs7-envelopedData
.. and encryptedData:
$ apps/openssl asn1parse -inform der -in fuzz/corpora/cms/2aba8037213156ea1593054768d4576cb8d083ed
    0:d=0  hl=2 l=  11 cons: SEQUENCE
    2:d=1  hl=2 l=   9 prim: OBJECT            :pkcs7-encryptedData
there was already test coverage for authEnvelopedData in fuzz-corpora/asn1
$ apps/openssl asn1parse -inform der -in fuzz-corpora/asn1/0a1a7d44160a74df6f1f9502b00ea6de8b9ead9e
    0:d=0  hl=2 l=inf  cons: SEQUENCE
    2:d=1  hl=2 l=  11 prim: OBJECT            :id-smime-ct-authEnvelopedData
   15:d=1  hl=3 l=   0 cons: cont [ 0 ]